gradio/blocks.py

```diff
@@ -2450,6 +2452,7 @@ def reverse(text):
                     block.key = f"__{block._id}__"
 
         self.pwa = utils.get_space() is not None if pwa is None else pwa
+        self.pwa_icon = pwa_icon
```

Comment:
In the original comment, the user specifically asks for the pwa icon to reflect the value of `favicon_path`. It would be cumbersome to expect users to put the same path twice for two separate parameters. I think we should just set the pwa_icon to be the favicon and remove the pwa_icon parameter.

Reply:
Agreed, if we get request for further customization, then I suggest we let developers specify a complete `manifest.json` file. Anything else seems too specific of a parameter.

Reply:
Thank you, that's correct.
I updated the code.